lib/svg-icon-registry.service.ts

```diff
@@ -51,4 +69,16 @@ export class SvgIconRegistryService {
 		}
 	}
 
+	// Extract sprite
+	getSprite(svg: SVGElement, symbolID: string) {
+		const svgElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+		const spriteElement: SVGSVGElement = svg.querySelector(`#${symbolID}`);
+		svgElement.setAttribute('width', spriteElement.viewBox.baseVal.width + 'px');
+		svgElement.setAttribute('height', spriteElement.viewBox.baseVal.height + 'px');
```

Comment:
Setting width and height circumvents width/height styling with CSS.  So, please remove these two lines of code. 

If setting width and height is desirable, then maybe an additional parameter in the component to instruct setting the viewbox width and height as the size.